publishers:
  butia_hotword_detection:
    topic: "/butia_speech/bhd/detected"

subscribers:
  butia_hotword_detection:
    topic: "/butia_speech/bhd/hot_word"

  speech_synthesizer:
    topic: "/butia_speech/ss/say_something"

  stream_data:
    topic: "/butia_speech/ap/stream_data"

services:
  speech_recognizer:
    service: "/butia_speech/sr/speech_recognizer"

  speech_sythesizer:
    service: "/butia_speech/ss/say_something"

  audio_player:
    service: "/butia_speech/ap/audio_player"

  audio_player_by_data:
    service: "/butia_speech/ap/audio_player_by_data"

  stream_start:
    service: "/butia_speech/ap/stream_start"

  stream_stop:
    service: "/butia_speech/ap/stream_stop"

warnings: False
